### PR TITLE
please merge pls pls

### PR DESCRIPTION
**Pull request by delta2571**

### CODE REVIEW

**Comment by m0thman70** [comment]
i would if i could

**Review by m0th05 — APPROVED** [review]
I saw arch install scripts being added but i suppose for now that's fine just make sure to change that

**Review by m0th05 — APPROVED** [review]
Looks good, i think this should be merged with main

**Comment by electron271** [comment]
do not remove xfce and add gnome until we have a consensus

**Review by electron271 — CHANGES_REQUESTED** [review]
see above

**Comment by electron271** [comment]
https://github.com/delta2571/GrainOS/pull/4

File: archiso/packages.x86_64 [hunk 1]
```diff
@@ -1,20 +1,20 @@
-#arch-install-scripts
-#archinstall
-#kitty-terminfo
+alacritty
```
Comment (electron271) [inline archiso/packages.x86_64]: suggestion: kitty

**Pull request merged**